src/python/WMCore/WorkQueue/Policy/Start/StartPolicyInterface.py

```diff
@@ -138,7 +138,12 @@ def newQueueElement(self, **args):
         ele = WorkQueueElement(**args)
         for data, sites in viewitems(ele['Inputs']):
             if not sites:
-                raise WorkQueueWMSpecError(self.wmspec, 'Input data has no locations "%s"' % data)
+                # make call to rucio to get list of replicas for this input data
+                locations = self.getDatasetLocations([data])
+                # we comment out raising exception due to issue-11784 and allow WQ element creation
+                # but we would like to monitor when and how often it happens
+                self.logger.warning('Input data has no location, spec=%s, dataset=%s, locations=%s', self.wmspec, data, locations)
```

Comment:
I would suggest to rename `dataset` by something more generic, e.g. "data" or "input data". Reason is, most of the times this will correspond to a block instead of a dataset.

Reply:
done